src/test/rustdoc/issue-85095.rs

```diff
@@ -0,0 +1,22 @@
+use std::ops::Deref;
```

Comment:
Can you give this file a descriptive name instead of the issue number? "recursive-deref" or something is fine.